dll/win32/browseui/addressband.cpp

```diff
@@ -259,8 +259,63 @@ HRESULT STDMETHODCALLTYPE CAddressBand::HasFocusIO()
     return S_FALSE;
 }
 
+static WCHAR GetAccessKeyFromText(WCHAR chAccess, LPCWSTR pszText)
+{
+    for (const WCHAR *pch = pszText; *pch != UNICODE_NULL; ++pch)
+    {
+        if (*pch == L'&' && pch[1] == L'&')
+        {
+            ++pch;
+            continue;
```

Comment:
For @JoachimHenze : In this first if-block, because the pch substring will be of the form: "&&..." (where the dots can denote any other characters, or none) and is NULL-terminated by assumption (as it was provided to the function), there will be two increments of pch pointer (first one here, second one in the for-loop), so that after that, pch will point to those "other characters" (before the terminating NULL), or none i.e. will point on the terminating NULL. In either of these cases this is OK. (Second case will terminate the for-loop.)